helpers/joinPath.js

```diff
@@ -1,4 +1,10 @@
 module.exports = function joinPath (/*base, path1, path2, ... */) {
+
+  if( ! arguments[1] || ! 	Array.isArray(arguments[1]) ){
+  	console.warn( "arguments[1] dependent must be an array" );
```

Comment:
add a prefix to warning, like `cerebral-module-forms: ` and make it less cryptic.
can you provide `jionPath` usage example to get an idea of how it works?